### PR TITLE
add primary keys to client counts

### PR DESCRIPTION
**Pull request by rebecca-burwei**

### CODE REVIEW

File: firefox_desktop/views/client_counts.view.lkml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+include: "//looker-hub/firefox_desktop/views/client_counts.view"
+view: +client_counts {
```
Comment (wwyc) [inline firefox_desktop/views/client_counts.view.lkml:2]:
```suggestion

view: +client_counts {
```

File: firefox_desktop/views/client_counts.view.lkml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  dimension: primary_key {
+    hidden: yes
+    primary_key: yes
+    sql: CONCAT(${TABLE}.client_id, ${TABLE}.submission_date) ;;
+    }
+
```
Comment (wwyc) [inline firefox_desktop/views/client_counts.view.lkml:8]:
```suggestion
```

File: firefox_desktop/views/client_counts.view.lkml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+view: +clients_daily_table__scalar_parent_urlbar_picked_autofill_adaptive_sum {
+  dimension: primary_key {
+    hidden: yes
+    primary_key: yes
+    sql: CONCAT(${client_counts.client_id}, ${client_counts.submission_date}) ;;
+  }
+
```
Comment (wwyc) [inline firefox_desktop/views/client_counts.view.lkml:17]:
```suggestion
```

**Review by wwyc — APPROVED** [review]
r+wc

File: firefox_desktop/views/client_counts.view.lkml (hunk 2 above)
Comment (rebecca-burwei) [reply firefox_desktop/views/client_counts.view.lkml:10]: I think the spacing is fixed now

File: firefox_desktop/views/client_counts.view.lkml (hunk 3 above)
Comment (rebecca-burwei) [reply firefox_desktop/views/client_counts.view.lkml:42]: I think the spacing is fixed now

**Pull request merged**